config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,52 @@
+# Throttle by IP address and by provided email address to 1 rpm on any contact forms
+# endpoint.
+
+# Is this likely to affect the accessible request form if a lot of errors?
```

Comment:
Can we remove this comment as the RAF form ends up being a non-throttled path?

Reply:
Yes, we can.